src/ffi/mod.rs

```diff
@@ -719,6 +739,11 @@ pub extern "C" fn telio_connect_to_exit_node_postquantum(
     allowed_ips: *const c_char,
     endpoint: *const c_char,
 ) -> telio_result {
+    telio_log_info!(
```

Comment:
Do we need this?

Reply:
We would like to log all FFI interactions in user logs, especially state-changing interactions, because this is first line of "debugging", when receiving logs. When receiving logs, it is providing the scenario on what kind of "scenario" was running. Therefore yes, PQ is needed here as well.